Library/src/Multiplayer/SpaceEntity.cpp

```diff
@@ -1020,6 +1020,9 @@ void SpaceEntity::ApplyLocalPatch(bool InvokeUpdateCallback)
 			{
 				if (Components.HasKey(TransientDeletionComponentIds[i]))
 				{
+					ComponentBase* Component = GetComponent(TransientDeletionComponentIds[i]);
+					Component->OnLocalDelete();
```

Comment:
Do we need to await a response from deleting the sequences before moving on? Might be fine to let it happen async.

Reply:
Possibly, I am getting a strange crash around this so I will try with waiting for a response